tracks/rust/exercises/atbash-cipher/mentoring.md

```diff
@@ -0,0 +1,164 @@
+# Atbash Cipher
+
+## Concepts
+
+- ascii
+- chars
+- iterators
+- primative types
+
+## Reasonable solutions
+
+A reasonable solution should do the following:
+
+- Use byte literals rather than integer literals to represent ascii characters.
+  Use `b'a'` rather than `97`.
+- Use char methods for working with ascii data, all of which contain `_ascii_`
+  in their name, rather than filtering characters with `is_ascii` and using
+  unicode centric methods for manipulation.
+- Factor the common logic into a seperate function rather than duplicating
+  filtering and transposition in encode and decode. **Bonus** its fine if this
+  function returns a String or if they just use decode as the starting point of
+  encode, but this is good place to introduce the `impl Trait` in the return
+  position after having already approved the exercise.
+- **Optional**: Avoid heap allocations when inserting spaces into the encoded
+  final string. Its better to `flat_map` spaces into the than collecting each
```

Comment:
“into the than” reads a bit weird for me. Can you rephrase that a bit, making sure it reads like a proper sentence for non-native speakers as well?

Reply:
It looks like its missing a word or two.